### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Malavi1**
I make an updated README file with added emoji reactions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,49 +19,49 @@ You can follow our twitter account
...
 
 ## Deployment
 
-Currently this application can be deployed as docker container or it can be deployed as standalone cron job
+Currently this application can be deployed as docker container or it can be deployed as standalone cron job.
 
 - [Docker Deployment](DOCKER_DEPLOYMENT.md)
 
-## Troubleshooting
+## Troubleshooting😵
```
Comment (arshadkazmi42) [inline README.md:35]:
```suggestion
## Troubleshooting 😵
```

File: README.md [hunk 2]
```diff
@@ -19,49 +19,49 @@ You can follow our twitter account
...
 or simply
-[![Say Thanks!](https://img.shields.io/badge/Say%20Thanks-!-1EAEDB.svg)](https://saythanks.io/to/arshadkazmi42&first-issues).
+[![Say Thanks!](https://img.shields.io/badge/Say%20Thanks-!-1EAEDB.svg)](https://saythanks.io/to/arshadkazmi42&first-issues)
 
 </div>
   
-## How to use
-Read our how to use guide [here](HOW_TO_USE.md).
+## How to use 🤔
+Read our how to use guide 👉[here](HOW_TO_USE.md).
```
Comment (arshadkazmi42) [inline README.md:27]:
```suggestion
Read our how to use guide [here](HOW_TO_USE.md).
```

File: README.md [hunk 3]
```diff
@@ -19,49 +19,49 @@ You can follow our twitter account
...
-Currently this application can be deployed as docker container or it can be deployed as standalone cron job
+Currently this application can be deployed as docker container or it can be deployed as standalone cron job.
 
 - [Docker Deployment](DOCKER_DEPLOYMENT.md)
 
-## Troubleshooting
+## Troubleshooting😵
 
-Seeing any errors while running the bot? Check out the [Troubleshooting Guide](TROUBLESHOOTING.md).
+Seeing any errors while running the bot? Check out the 🤠[Troubleshooting Guide](TROUBLESHOOTING.md).
```
Comment (arshadkazmi42) [inline README.md:37]:
```suggestion
Seeing any errors while running the bot? Check out the [Troubleshooting Guide](TROUBLESHOOTING.md).
```

File: README.md [hunk 4]
```diff
@@ -19,49 +19,49 @@ You can follow our twitter account
...
 
-If your problem is not present there, then go ahead and create an [issue](https://github.com/arshadkazmi42/first-issues/issues/new)
+If your problem is not present there,🧐 then go ahead and create an🤩 [issue](https://github.com/arshadkazmi42/first-issues/issues/new)
```
Comment (arshadkazmi42) [inline README.md:39]:
```suggestion
If your problem is not present there, then go ahead and create an [issue](https://github.com/arshadkazmi42/first-issues/issues/new)
```

File: README.md [hunk 5]
```diff
@@ -19,49 +19,49 @@ You can follow our twitter account
...
 
 ## Contributing
 
-- If you have got any suggestions, drop your suggestions [here](https://github.com/arshadkazmi42/first-issues/issues/new).
+- If you have got any suggestions, drop your suggestions😇 [here](https://github.com/arshadkazmi42/first-issues/issues/new).
```
Comment (arshadkazmi42) [inline README.md:43]:
```suggestion
- If you have got any suggestions, drop your suggestions [here](https://github.com/arshadkazmi42/first-issues/issues/new).
```

File: README.md [hunk 6]
```diff
@@ -19,49 +19,49 @@ You can follow our twitter account
...
 - If you want to fix some existing bugs in the script, you can find open issues [here](https://github.com/arshadkazmi42/first-issues/issues)
 
-Read our Contribution guide [here](Contribution.md).
+Read our Contribution guide 🤓 [here](Contribution.md).
```
Comment (arshadkazmi42) [inline README.md:46]:
```suggestion
Read our Contribution guide [here](Contribution.md).
```

File: README.md [hunk 7]
```diff
@@ -19,49 +19,49 @@ You can follow our twitter account
...
 
-## Sponsors
+## Sponsors🥳
```
Comment (arshadkazmi42) [inline README.md:48]:
```suggestion
## Sponsors 🥳
```

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
I think we should keep emojis only in the titles. Including emojis in the text is making it look cluttered.

But I like the emojis in the title.

Can you remove other emojis and keep the ones with the titles. Also I have added few review comments about spacing

**Comment by Malavi1** [comment]
That is also fine !! I remake this by placing emojis in the titles

**Comment by arshadkazmi42** [comment]
@Malavi1 These emojis looks good. You can update this PR by editing files [here](https://github.com/arshadkazmi42/first-issues/pull/107/files)

You can make the changes for the review comment I have added. Once those are done we can merge this pull request

**Comment by Malavi1** [comment]
In addition to the requested changes I replaced a few things also ! if  you thing good then continue otherwise I also ready to make changes 


thankyou !!!

**Review by arshadkazmi42 — APPROVED** [review]
Looks good. Thanks

**Comment by Malavi1** [comment]
My pleasure !!!

**Pull request merged**